docs/rfcs/RFC0002-MonoRepo-Gardener-Rotations.md

```diff
@@ -0,0 +1,137 @@
+# Mono-repo Gardener
+
+Gardeners<sup>[1]</sup> are responsible for keeping their mono-repos shippable;
+ensuring bad commits get reverted by keeping build, test, and nightly release
+workflows green so that developers can remain productive and bugs are caught
+closer to commit times.
+
+In order to facilitate this work, gardeners have some super powers afforded to them
```

Comment:
Such as:

* The ability to bypass any checks in the CI.
* Authority to request that any developer on the project who is implicated in a breakage/degradation of service/resolution stop what they are doing and participate.
* In the event of major breakages, the ability to escalate and pull higher level management in to resolve issues.